deploy/arbitrum/250_deploy_child_gauges.ts

```diff
@@ -16,6 +16,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     SaddleFRAXUSDTMetaPoolLPToken: "FRAXBP-USDT",
     SaddleFRAXUSDsMetaPoolLPToken: "FRAXBP-USDs",
     SaddleUSXFRAXBPMetaPoolLPToken: "USX-FRAXBP",
+    CommunityfUSDCPoolLPToken: "fUSDC-USDC",
```

Comment:
Registry mismatch will lead to mismatched deployment address of Root/Child. Should use same name mapping on both mainnet and sidechain.